packages/Skipper/Skipper/Skipper.php

```diff
@@ -23,16 +22,13 @@ final class Skipper
     private const FILE_ELEMENT = 'file_elements';
 
     /**
-     * @var SkipVoterInterface[]
+     * @param iterable<SkipVoterInterface> $skipVoters
      */
-    private array $skipVoters = [];
-
     public function __construct(
-        ClassSkipVoter $classSkipVoter,
-        PathSkipVoter $pathSkipVoter,
-        private readonly RectifiedAnalyzer $rectifiedAnalyzer
+        private readonly RectifiedAnalyzer $rectifiedAnalyzer,
+        private readonly iterable $skipVoters = [],
```

Comment:
This should be non-empty, to avoid missing the default skip voters:

```suggestion
        private readonly array $skipVoters,
```